lib/licensee/project_file.rb

```diff
@@ -1,18 +1,18 @@
+require 'charlock_holmes'
+
 module Licensee
   class Project
     class File
       attr_reader :content, :filename
 
-      ENCODING = Encoding::UTF_8
-      ENCODING_OPTIONS = {
-        invalid: :replace,
-        undef:   :replace,
-        replace: ''
-      }.freeze
-
       def initialize(content, filename = nil)
         @content = content
-        @content.encode!(ENCODING, ENCODING_OPTIONS)
+        unless @content.empty?
```

Comment:
That was the hint I needed to avoid Charlock entirely. Thanks! Alternative in #200